common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -263,7 +266,7 @@ def has_support(self, view, functionality):
         Returns whether the given view has support for the given functionality.
         """
         if functionality == "multi_device":
-            return all(
+            return self.problem_types and all(
```

Comment:
I wasn't 100% sure of what the desired logic should be here - if there are no problem types, should `has_support` return False, as I've done here? Or is this logic more accurately checking "there are no problem types that *don't* support multi-device"?